src/Header/AbstractAccept.php

```diff
@@ -170,7 +170,7 @@ protected function getParametersFromFieldValuePart($fieldValuePart)
             foreach ($paramsStrings as $param) {
                 $explode = explode('=', $param, 2);
 
-                if (count($explode) === 2) {
+                if (isset($explode[1])) {
```

Comment:
Should check that key 0 exists, and key 2 doesn't.

Reply:
The explode operation was given a limit of 2, so there's no chance a key 2 will ever exist.